pkg/collector/corechecks/network-devices/versa/client/client.go

```diff
@@ -612,6 +612,73 @@ func (client *Client) GetLinkUsageMetrics(tenant string) ([]LinkUsageMetrics, er
 	return metrics, nil
 }
 
+// parseTunnelMetrics parses the raw AaData response into TunnelMetrics structs
+func parseTunnelMetrics(data [][]interface{}) ([]TunnelMetrics, error) {
+	var rows []TunnelMetrics
+	for _, row := range data {
+		m := TunnelMetrics{}
+		// Based on the new structure, we expect 6 columns
```

Comment:
7 columns?

Reply:
Ah, yes. I added a field quickly and forgot to update these. Thanks for pointing it out!